/home/s1572869/miniconda3/etc/profile.d/conda.sh: /home/s1572869/miniconda3/bin/conda: /home/s1572869/miniconda3/bin/python: bad interpreter: Stale file handle
Traceback (most recent call last):
  File "cifar100_experiments.py", line 8, in <module>
    from lib.experiments import load_dataset, run_wcl, train_with_original, run_pop, run_egdis, run_cl, run_wcl2, run_wcl3
  File "/home/s1572869/MSc_results/lib/experiments.py", line 15, in <module>
    from lib.data_loader import load_cifar10, load_cifar100
  File "/home/s1572869/MSc_results/lib/data_loader.py", line 26, in <module>
    from tensorflow.python.keras import backend as K
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow/__init__.py", line 101, in <module>
    from tensorflow_core import *
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow_core/__init__.py", line 40, in <module>
    from tensorflow.python.tools import module_util as _module_util
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow/__init__.py", line 50, in __getattr__
    module = self._load()
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow/__init__.py", line 44, in _load
    module = _importlib.import_module(self.__name__)
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow_core/python/__init__.py", line 64, in <module>
    from tensorflow.core.framework.graph_pb2 import *
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/tensorflow_core/core/framework/graph_pb2.py", line 7, in <module>
    from google.protobuf import descriptor as _descriptor
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/google/protobuf/descriptor.py", line 48, in <module>
    from google.protobuf.pyext import _message
ImportError: libprotobuf.so.23: cannot open shared object file: No such file or directory
